src/Persistence/YamlValidator/ValidatorFactory.cs

```diff
@@ -6,9 +6,11 @@
 
 namespace Microsoft.PowerPlatform.PowerApps.Persistence.YamlValidator;
 
-public class ValidatorFactory
+internal class ValidatorFactory : IValidatorFactory
 {
-    public Validator GetValidator()
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static",
```

Comment:
I personally feel like this should be a suggestion, and not enforced. but there are performance reasons for it.
But, I believe a better Justification would be "required by IValidatorFactory interface"